appveyor.yml

```diff
@@ -152,6 +152,10 @@ install:
   # Attempt to debug sccache failures
   - set SCCACHE_ERROR_LOG=%CD%/sccache.log
 
+before_build:
+  # Run unit tests for bootstrap.py
+  - python src\bootstrap\bootstrap_test.py
+
```

Comment:
Ok I've had a look at this again and realised I missed something - we run Docker containers for a number of our builds and so this python script should ideally run in those as well.

My suggestion is to remove the additions to the travis and appveyor yml, and instead mimic https://github.com/rust-lang/rust/blob/3447aa7/src/ci/run.sh#L65-L69 - hopefully that will work!